admin/partials/admin-monk-select-menu-to-edit-render.php

```diff
@@ -77,7 +77,7 @@
 <?php
 $menu_id           = empty( $menu ) || 'delete' === filter_input( INPUT_GET, 'action' ) ? get_user_option( 'nav_menu_recently_edited' ) : $menu;
 
-if ( $menu_id ) :
+if ( get_term( $menu_id ) ) :
```

Comment:
Check `get_term` documentation and make a fix on this conditional.
This function has multiple return types: **(mixed | null | WP_Error)**

So if it's a `WP_Error`, you'll continue the process.